pki.spec

```diff
@@ -1482,6 +1490,12 @@ fi
 # with_python3_default
 %endif
 
+%{_bindir}/pki-healthcheck
```

Comment:
Should this be `%{_sbindir}` since the tool requires root privilege like the other server tools?

Reply:
The `python setup.py install` creates the executable at `<prefix>/usr/bin/pki-healthcheck`. So, i ended up placing at the same location

Reply:
Seems like it can be placed into the sbin directory. I have updated in my commit